src/include/ZividPython/ReleasableArray2D.h

```diff
@@ -1,9 +1,11 @@
 #pragma once
 
-#include <Zivid/PointCloud.h>
 #include <ZividPython/Releasable.h>
 #include <ZividPython/Wrappers.h>
 
+#include <Zivid/Color.h>
+#include <Zivid/PointCloud.h>
```

Comment:
PointCloud.h already includes Color.h via Image.h, so I don't think this change is necessary.

Reply:
Ah, clangd decided to add it for me. Its not necessary, but maybe its okay to have it explicit?

Reply:
Sure, no problem.